builder/bundles.go

```diff
@@ -624,7 +624,7 @@ func (b *Builder) buildBundles(set bundleSet, downloadRetries int) error {
 	outputDir := filepath.Join(b.Config.Builder.ServerStateDir, "www")
 
 	if _, ok := set["os-core"]; !ok {
-		return fmt.Errorf("os-core bundle not found")
+		fmt.Printf("Warning: os-core bundle not found. System may not be complete.\n")
```

Comment:
I don't like having a warning for something that is expected behavior. I'd rather remove it or have it print some sort of "build mode" message, say "Building package mix"/"Building system mix". Those are not proper terms we use in mixer, so we might as well just remove the warning.

Reply:
ok. I'll remove.
Is there a problem on now enforcing os-core? Should I add a flag or config to make sure no one will create a regular mix without os-core?